samples/alarmbot-es6-botframework-webchat/src/alarmRenderer.js

```diff
@@ -0,0 +1,11 @@
+export class AlarmRenderer {
```

Comment:
I thought the norm in a vanilla JS project was to just put all of the code in "./lib"? The idea being that since you don't require a compile step your JS is already in output form.